osf/metadata/gather/gatherer.py

```diff
@@ -61,11 +61,16 @@ def add_gatherer(gatherer, predicate_iris, focustype_iris):
         )
 
 
-def get_gatherers(focustype_iri, predicate_iris):
+def get_gatherers(focustype_iri, predicate_iris, *, include_focustype_defaults=True):
```

Comment:
What is this `*` in the parameters?

Reply:
it restricts any following parameters as keyword-only
```py
>>> def foo(*, blah):
...     print(blah)
... 
>>> foo(2)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: foo() takes 0 positional arguments but 1 was given
>>> foo(blah=2)
2
```
likewise, can use `/` to restrict any preceeding parameters as positional-only -- see https://docs.python.org/3.13/tutorial/controlflow.html#special-parameters

(this lets me encode my bias ("stylistic preference"?) that passing unnamed boolean flags like `some_function(True, True, False)` is hostile to the reader)